pkg/TIPA.py

```
@@ -519,23 +532,23 @@ def correct_text(self, wordlist: list) -> str:
         if not search_text:
             return None
-        
+
         base_urls = {
             'market': 'https://www.google.com/search',
             'wiki': 'https://www.google.com/search'
         }
-        
+
         if site not in base_urls:
             raise ValueError(f"Invalid site: {site}")
```

Ruff (0.8.0)

Avoid specifying long messages outside the exception class

(TRY003)